### PR TITLE
added static data classes

### PR DESCRIPTION
**Pull request by collinnolen**

### CODE REVIEW

File: Assets/Scripts/Combat/CombatManager.cs [hunk 1]
```diff
@@ -12,8 +12,8 @@ public class CombatManager : MonoBehaviour {
 
     private GameManager gM;
 
-    private StaticCharacterData playerStaticData;
-    private StaticCharacterData enemyStaticData;
+    //private StaticCharacterData playerStaticData;
```
Comment (artsiriwatt) [inline Assets/Scripts/Combat/CombatManager.cs]: Just delete - we can reference previous lines via git commit history

File: Assets/Scripts/Combat/LiveCombatData.cs [hunk 2]
```diff
@@ -35,16 +35,16 @@ public LiveCombatData(bool isTest)
...
-    {
-        this.currentShipHP = scd.currentShipHP;
-        this.maxShipHP = scd.maxShipHP;
-        this.currentCrewHP = scd.currentCrewHP;
-        this.maxCrewHP = scd.maxCrewHP;
-        this.manaRegen = scd.manaRegen;
-        this.maxMana = scd.maxMana;
-        this.handSize = scd.maxHandSize;
-    }
+    //public void GatherData(StaticCharacterData scd)
```
Comment (artsiriwatt) [inline Assets/Scripts/Combat/LiveCombatData.cs]: Leave the signature blank ( i.e. GatherData() ) and leave a TODO comment that this needs implementation in the body of the method

File: Assets/Scripts/shared/StaticEnemies.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+public static class StaticEnemies {
+
+    static List<EnemyData> list;
+
+    public static void UpdateEnemyDataById(string id)
```
Comment (artsiriwatt) [inline Assets/Scripts/shared/StaticEnemies.cs]: I would say we need an "add enemyData/remove enemyData" method, but updating specific enemies should be done via "GetEnemyDataById" and altering that EnemyData versus having a specific method for it here

**Comment by artsiriwatt** [comment]
With this being time sensitive, i'll take a pass on this

**Comment by artsiriwatt** [comment]
took a pass on this; I was able to manually test by manually manipulating a value on StaticPlayer, switching scenes (via hard coding the scene switch), and printing out the changes/updated value

**Comment by collinnolen** [comment]
Approved

**Pull request merged**